wordpress/wp-content/themes/cds-default/functions.php

```diff
@@ -87,6 +91,59 @@ function cds_setup(): void
             'style',
             'script',
         ]);
+
+        add_theme_support('editor-color-palette', array(
+            array(
+                'name'  => __('Black', 'cds-snc'),
+                'slug'  => 'black',
+                'color' => '#000000',
+            ),
+            array(
+                'name'  => __('White', 'cds-snc'),
+                'slug'  => 'white',
+                'color' => '#ffffff',
+            ),
+            array(
+                'name'  => __('Primary Blue', 'cds-snc'),
+                'slug'  => 'primary-blue',
+                'color' => '#284162',
+            ),
```

Comment:
Just FYI, there are two Canada.ca blue colours:

- ![#26374a](https://via.placeholder.com/15/26374a/000000?text=+) `#26374a` for buttons
- ![#284162](https://via.placeholder.com/15/284162/000000?text=+) `#284162` for link text

It's a Very Cool Feature of WET, as far as I can tell.

Reply:
We should add the other one for sure + come up with better names and add missing colours to the palette  --- going to merge this one but for sure want to circle back.

There's also another way to select buttons which might be coming from the Bootstrap blocks plugin (not sure).  I would like the user to edit as need be and or pick a solid default.

<img width="316" alt="Screen Shot 2021-11-01 at 2 41 52 PM" src="https://user-images.githubusercontent.com/62242/139723833-b4b3ae81-8d06-4e10-a4ae-b12a0302c961.png">